### ISSUE TITLE
add accessibility attributes to stateless template

### ISSUE THREAD

**Issue by bragamat** [bragamat/leviosajs#15]
Let's try to always keep accessibility first! In that case I've experienced lack of some attributes as especifications. 

- [ARIA](https://developer.mozilla.org/pt-BR/docs/Web/Accessibility/ARIA) [pt-BR]
- [mozila](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML)
- [w3s](https://www.w3schools.com/html/html_accessibility.asp)
